src/encoder.js

```diff
@@ -65,9 +65,8 @@ Encoder.prototype.getValue= function(value,type){
  */
 function sanitizeData(data) {
     if(typeof data === "string"){
-        appCharsArr.forEach(c => {
-            data = data.replace(new RegExp(c, 'g'), '\\'+c);
-        })
+        let regex = new RegExp(appCharsArr.join('|'), 'g');
```

Comment:
Do we need to create the regex on every function call? I believe it'll impact the performance too.